### PR TITLE
NR-56439-Snowflake-quickstart

### PR DESCRIPTION
**Pull request by pkudikyala**
# Summary

**Ticket URL:** https://issues.newrelic.com/browse/NR-56439

**Description:** This PR has the new quickstart integration named "Snowflake"

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include an InstallPlan and Documentation reference?
- [ ] Are all documentation links publically accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

# Screenshot

screencapture-newrelic-instant-observability-preview-2022-11-09-12_57_20

### CODE REVIEW

File: install/snowflake/install.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+target:
+  type: integration
+  destination: host
+  os:
+    - linux
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/
```
Comment (josephgregoryii) [inline install/snowflake/install.yml:15]: Looks like we need this URL to be updated with a specific link 😄

File: quickstarts/snowflake/config.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+level: New Relic
+authors:
+  - New Relic
+title: Snowflake
+documentation:
+  - name: Snowflake integration documentation
+    description: |
+      Quickly monitor your Snowflake application by tracking performance issues with a full end-to-end view of distributed traces 
+    url: >-
+      https://docs.newrelic.com/
```
Comment (josephgregoryii) [inline quickstarts/snowflake/config.yml:42]: Here as well

**Review by josephgregoryii — COMMENTED** [review]
Hey @pkudikyala - looks great! just 2 comments about the documentation URLs below and we can go ahead and merge it in

File: install/snowflake/install.yml (hunk 1 above)
Comment (pkudikyala) [reply install/snowflake/install.yml:15]: Hi @josephgregoryii,
Yes, but they are not yet ready with the installation docs. Once the docs are ready I will update them and move to review.

Thank you...

File: install/snowflake/install.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+target:
+  type: integration
+  destination: host
+  os:
+    - linux
+
+install:
+  mode: link
+  destination:
+    url: https://newrelic.com/instant-observability/preview?local=true&quickstart=snowflake
```
Comment (pkosanam) [inline install/snowflake/install.yml:15]: pls update this link with the docs link

**Review by pkosanam — CHANGES_REQUESTED** [review]
please add docs link on dashboard too

File: install/snowflake/install.yml (hunk 3 above)
Comment (pkudikyala) [reply install/snowflake/install.yml]: Updated doc's url

**Review by pkosanam — APPROVED** [review]

**Review by josephgregoryii — APPROVED** [review]
Looks good! thanks

**Pull request merged**